src/localstore/incremental/JasmineGraphIncrementalLocalStore.h

```diff
@@ -0,0 +1,16 @@
+#include <string>
```

Comment:
Add license header

Reply:
Fixed in https://github.com/miyurud/jasminegraph/pull/122/commits/bdd734e88390169ddc9e44c74030d1c1b170d0ae

Reply:
Done